semanticcpg/src/main/scala/io/shiftleft/semanticcpg/passes/languagespecific/fuzzyc/TypeDeclStubCreator.scala

```diff
@@ -50,8 +56,8 @@ class TypeDeclStubCreator(cpg: Cpg) extends CpgPass(cpg) {
   }
 
   private def init(): Unit = {
-    cpg.typeDecl.sideEffect { typeDecl =>
+    cpg.typeDecl.l.par.foreach { typeDecl =>
```

Comment:
Did you measure whether this improves performance? I would be very surprised if it does but that is just a feeling.